modules/tinymce/src/core/main/ts/api/dom/Selection.ts

```diff
@@ -84,6 +91,7 @@ interface EditorSelection {
   getStart: (real?: boolean) => Element;
   getEnd: (real?: boolean) => Element;
   getSelectedBlocks: (startElm?: Element, endElm?: Element) => Element[];
+  getSelectedCells: () => HTMLTableCellElement[];
```

Comment:
What's going on here? We said this should not exist.

- [ ] Remove and merge into `getSelectedBlocks` as discussed or come up with some other solution

Reply:
I have added a TODO to look into this and try and come up with another better solution

Reply:
Luke showed me his attempt to combine get selected cells with get selected blocks, it became a huge mess. I believe he mentioned it around the time he opened this PR but there's been a lot of chat back and forth so I'm not surprised you missed it.

I'd like to get this merged and then take a step back and see what options we have for combining them.

Reply:
Yeah that's fine, I'd mentioned to Luke to just log a follow up Jira on Friday but forgot to update this sorry.